<?xml version="1.0"?>
<launch>

	<node pkg="nodelet" type="nodelet" name="cpc_nodelet"  args="manager"/>

	<node pkg="nodelet" type="nodelet" name="PcPublisher" args="load cob_vision_ipa_utils/PcPublisher cpc_nodelet">
		<remap from="image_xyz" to="cob_pcl/fused_image_xyz"/>
		<remap from="image_color" to="cob_pcl/fused_image_color"/>
		<remap from="image_confidence" to="cob_pcl/fused_image_confidence"/>
		<remap from="image_features" to="cob_pcl/fused_image_features"/>
		<!--remap from="point_cloud" to="cpc_publisher/point_cloud"/-->
		<!--remap from="image_xyz" to="sensor_fusion/shared/image_xyz"/>
		<remap from="image_color" to="sensor_fusion/shared/image_color"/>
		<remap from="image_confidence" to="sensor_fusion/shared/image_confidence"/-->
	</node>
	
	<node pkg="nodelet" type="nodelet" name="CPCVisualizer" args="load cob_vision_ipa_utils/CPCVisualizer cpc_nodelet">
		<remap from="point_cloud_filtered" to="cob_pcl/point_cloud_filtered"/>
	</node>
	
	<!--node pkg="nodelet" type="nodelet" name="VoxelGrid" args="load pcl/VoxelGrid cpc_nodelet">
	</node-->

</launch>
